utils/prometheusmetrics/prometheusmetrics.go

```diff
@@ -172,6 +172,20 @@ var (
 		},
 		[]string{"eni"},
 	)
+	IpamdStartupDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name: "awscni_ipamd_startup_duration_seconds",
+			Help: "The duration of IPAMD startup from process start to ready to serve CNI requests",
+		},
+		[]string{"success", "with_api_server", "failure_reason"},
+	)
+	IpamdStartupAwsDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name: "awscni_ipamd_startup_aws_duration_seconds",
```

Comment:
Can we name this something like "awscni_ipamd_node_initialization_seconds" ? We do more than just aws api calls..

Reply:
sg!